task/redhat-codeready-dependency-analysis/0.1/redhat-codeready-dependency-analysis.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+
+metadata:
+  name: redhat-codeready-dependency-analysis
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.19.0"
+    tekton.dev/tags: Security, Vulnenrability, CVE
+    tekton.dev/displayName: "RedHat CodeReady Dependency Analysis"
+
+spec:
+  description: >-
+    The RedHat CodeReady Dependency Analysis task is an interface between Tekton and Red Hat CodeReady Dependency Analytics platform.
+    It provides vulnerability and compliance analysis for your Python applications dependencies, along with recommendations to address security vulnerabilities and licensing issues.
+
+  workspaces:
+    - name: output
+      description: Volume backing this workspace is used for input/output of the task.
+
+  params:
+    - name: manifest-file-path
+      description: Path of target manifest file to perform analysis.
+      default: requirements.txt
+
+    - name: output-file-path
+      description: Path of the file to save analysis report.
+      default: redhat-codeready-dependency-analysis-report.json
+
+    - name: pkg-installation-directory-path
+      description: Path of a directory in workspace, where dependencies are installed.
+      default: site-package
+
+    - name: image
+      description: Image where CRDA CLI binary and required applications are installed.
+      default: ghcr.io/fabric8-analytics/crda-python:3.6
+
+    - name: CRDA_HOST_URL
+      description: The CRDA host, adjust this if you use an on premise environment.
+      default: ""
```

Comment:
can we document this in the README as well?

Reply:
Yes, I am yet to complete modifications for tests, there are some changes required in CRDA CLI as well as schema of CRDA API is fixed in CLI and one of the rules there is that API should have HTTPS protocol, but while we are faking API the host of API is set to **http://127.0.0.1:8080** which fails in CLI side, so these changes are being done in CLI and a new release will be used in this TASK, that is the reason I am holding this PR.